### DISCUSSION TITLE
Remove basic authentication support from the Artifactory provider

### DISCUSSION THREAD

**Discussion by danielmkn** [jfrog/terraform-provider-artifactory#350] [Ideas] [answered]
#342 
We think we need to remove basic authentication from the Artifactory Provider. Basic auth will be removed in the following Artifactory versions, so it's the right move. 
This change is a part of a core product roadmap. Tokens can be scoped per user. Please read more in the official documentation:
[Access token documentation](https://www.jfrog.com/confluence/display/JFROG/Access+Tokens)
How to generate an [Access token](https://www.jfrog.com/confluence/display/JFROG/Access+Tokens#AccessTokens-GeneratingExpirableTokens).
We want to know any ideas or concerns from the contributors' community! Please let us know what do you think.

**Comment by alexhung** [comment, marked as answer]
Basic auth is removed in [v4.0.0](https://github.com/jfrog/terraform-provider-artifactory/releases/tag/v4.0.0)
